attendance/serializers.py

```diff
@@ -1,17 +1,71 @@
+from datetime import datetime
+
 from rest_framework import serializers
 
 from attendance.models import Attendance, Leaves
 
 
 class AttendanceSerializer(serializers.ModelSerializer):
-
     class Meta:
         model = Attendance
         fields = ["id", "employee", "check_in", "check_out", "status"]
 
+    def to_representation(self, instance):
+        ret = super(AttendanceSerializer, self).to_representation(instance)
+        ret['employee_name'] = str(str(instance.employee.first_name).capitalize() + " " +
+                                   str(instance.employee.last_name).capitalize())
+        ret['time_check_in'] = str(str(instance.check_in.hour + 5).zfill(2) + ":" +
+                                   str(instance.check_in.minute).zfill(2) +
```

Comment:
treat date type fields as DateTime Field, and user parsers for that if possible.